static/app/views/dashboardsV2/widget/index.tsx

```diff
@@ -13,7 +13,7 @@ type Props = React.ComponentProps<typeof WidgetBuilder>;
 function WidgetBuilderContainer({organization, ...props}: Props) {
   return (
     <Feature
-      features={['metrics', 'dashboards-edit']}
+      features={['metrics', 'metrics-dashboards-ui', 'dashboards-edit']}
```

Comment:
Should we also update `dashboardsV2/dashboard.tsx`?

Reply:
oh yes sure! ty!